src/commands/src-file-extension-mapping.ts

```diff
@@ -1,20 +1,20 @@
 export default {
-  mjs: 'js',
-  jsx: 'js',
-  ts: 'js',
-  tsx: 'js',
-  vue: 'js',
-  svelte: 'js',
-  mdx: 'js',
-  svx: 'js',
-  elm: 'js',
-  yaml: 'json',
-  toml: 'json',
-  php: 'html',
-  md: 'html',
-  ejs: 'html',
-  njk: 'html',
-  scss: 'css',
-  sass: 'css',
-  less: 'css',
```

Comment:
This may be a controversial change: introducing starting `.` to this map.

I did so because we were using `path.extname(fileName).substr(1)` over and over and over again. It felt like changing it here reduces a lot of that, and makes things predictably start with a `.`.

The only danger is when parsing the `build:*` scripts, so that’s one of the few places `.substr(1)` is left.

Reply:
YES! see comment above, +1 for making this change throughout